src/main/java/org/joda/beans/gen/BeanCodeGen.java

```diff
@@ -91,21 +92,20 @@ public static BeanCodeGen createFromArgs(String[] args) {
         if (args == null) {
             throw new IllegalArgumentException("Arguments must not be null");
         }
-        String indent = "    ";
-        String prefix = "";
-        String eol = System.lineSeparator();
-        String defaultStyle = null;
-        boolean recurse = false;
-        boolean generatedAnno = false;
-        int verbosity = 1;
-        boolean write = true;
-        File file = null;
-        BeanGenConfig config = null;
+        var indent = "    ";

```

Comment:
@coderabbitai Using deep analysis based on Java 21, categorise each change in the PR as semantic or stylistic. Provide a list of up to 50 changes that are semantic.